src/ml_gym/frontend/dashboard/src/app/store.tsx

```diff
@@ -0,0 +1,17 @@
+import { configureStore, ThunkAction, Action, } from '@reduxjs/toolkit'
+import jobStatusReducer from '../features/jobsStatus/jobsStatusSlice'
+import modelStatusReducer from '../features/modelsStatus/modelsStatusSlice'
+import modelEvaluationReducer from '../features/modelEvaluations/modelEvaluationsSlice'
+
+
+export const store = configureStore({
+  reducer: {
+    jobsStatus: jobStatusReducer,
+    modelsStatus: modelStatusReducer,
+    modelsEvaluation: modelEvaluationReducer
+  }
+});
+
+export type AppDispatch = typeof store.dispatch;
+export type RootState = ReturnType<typeof store.getState>;
+export type AppThunk<ReturnType = void> = ThunkAction<ReturnType, RootState, unknown, Action<string>>;
```

Comment:
Codacy has a fix for the issue: [Replace `ReturnType,·RootState,·unknown,·Action<string>` with `⏎··ReturnType,⏎··RootState,⏎··unknown,⏎··Action<string>⏎`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
export type AppThunk<ReturnType = void> = ThunkAction<
```